src/models/cost_function_timeseries.jl

```diff
@@ -158,6 +158,59 @@ function set_variable_cost!(
     return
 end
 
+function _process_fuel_cost(::Component, fuel_cost::Float64, start_time, len)
```

Comment:
```suggestion
function _process_fuel_cost(::Component, fuel_cost::Float64, start_time::Dates.DateTime, len::Int)
```

Reply:
Addressed, though I think in a more flexible manner